pom.xml

```diff
@@ -126,7 +116,7 @@
     <dependency>
       <groupId>io.rest-assured</groupId>
       <artifactId>rest-assured</artifactId>
-      <version>4.3.1</version>
+      <version>5.1.0</version>
```

Comment:
In similar RMB upgrade PR's we are upgrading to 5.1.1, so we can use the same version here